build.zig

```diff
@@ -1,10 +1,11 @@
 const std = @import("std");
-const zig_libressl = @import("zig-libressl/build.zig");
+pub const zig_libressl = @import("zig-libressl/build.zig");
 const Pkg = std.build.Pkg;
 
 fn relativeToThis(comptime path: []const u8) []const u8 {
     comptime {
-        return std.fs.path.dirname(@src().file).? ++ "/" ++ path;
+        const fspath = std.fs.path;
+        return fspath.dirname(@src().file).? ++ fspath.sep_str ++ path;
```

Comment:
In the future I'd like to avoid local shortenings like this

Reply:
You mean writing out the full path? `std.fs.path` is long.

Reply:
Yes